collectors/ebpf.plugin/ebpf_disk.c

```diff
@@ -321,7 +321,7 @@ static int read_local_disks()
 {
     char filename[FILENAME_MAX + 1];
     snprintfz(filename, FILENAME_MAX, "%s%s", netdata_configured_host_prefix, NETDATA_EBPF_PROC_PARTITIONS);
-    procfile *ff = procfile_open(filename, " \t:", PROCFILE_FLAG_DEFAULT);
+    procfile *ff = procfile_open_no_log(filename, " \t:", PROCFILE_FLAG_DEFAULT);
```

Comment:
This is reading `/proc/partitions`. Why do you think this is an unnecessary error?

Reply:
This was a mistake, it is good to have an error here, because if kernel changes something,we need to have this kind of error.